### PR TITLE
use strcmp instead of == in addpath_drake

### PR DESCRIPTION
**Pull request by hongkai-dai**
I got an error in my linux machine since my computer('arch') is 'glnxa64', the length of the string is not equal to the length of 'maci64'

### CODE REVIEW

**Comment by RussTedrake** [comment]
yikes.  how did that get through the build servers?  thanks!

**Comment by peteflorence** [comment]
Thanks @hongkai-dai for fixing :+1:

File: drake/addpath_drake.m [hunk 1]
```diff
@@ -60,12 +60,12 @@
...
 
 % OSX platform-specific: check if reverted to IPv4
-if (computer('arch') == 'maci64')
+if (strcmp(computer('arch'),'maci64'))
   ipv4_preferred = java.lang.System.getProperty('java.net.preferIPv4Stack');
   if isempty(ipv4_preferred)
     ipv4_preferred = 'false';
   end
-  if (ipv4_preferred ~= 'true')
+  if (strcmp(ipv4_preferred,'true'))
```
Comment (rdeits) [inline drake/addpath_drake.m]: This fix is backwards; the previous version checked if ipv4_preferred was *not* equal to `'true'`, but this checks if it *is* equal

Comment (hongkai-dai) [inline drake/addpath_drake.m]: oops, will send a pull request again, thanks @rdeits

Comment (peteflorence) [inline drake/addpath_drake.m]: Hongkai I'm on it, also want to check for >= Yosemite version

**Pull request merged**